spec/openapi.yaml

```diff
@@ -53,182 +248,743 @@ paths:
+                        'paid': false,
+                        'times_clicked': 3,
+                        'upvotes': 1,
+                        'url': 'http://www.test.com'
+                      }
+                  ],
+                    'has_next': false,
+                    'has_prev': false,
+                    'number_of_pages': 1,
+                    'page': 1,
+                    'records_per_page': 20,
+                    'status': 'ok',
+                    'status_code': 200,
+                    'total_count': 1
+                }
+        404:
+          $ref: '#/components/responses/NotFound'
+    post:
+      summary: Create Resource
+      description: Creates a new learning resource entry. The required properties for a new entry are `category`, `name`, `paid`, and `url` and must be included in the request body.
+      tags:
+        - Create Resource
+      requestBody:
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/CreateResourceRequest'
+            example: |
+              {
+                    'category': 'Tutorial',
+                    'name': 'CSS Tutorial',
+                    'paid': false,
+                    'url': 'https://www.w3schools.com/css/'
+                }
+        required: true
+      responses:
+        200:
+          description: Created resource
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/Error'
+                $ref: '#/components/schemas/Resource'
+              example: |
+                    {
+                        'apiVersion': '1.0',
+                        'data': {
+                            'category': 'Tutorial',
+                            'created_at': '2019-07-21 15:41:19',
+                            'downvotes': 0,
+                            'id': 2137,
+                            'languages': [],
+                            'last_updated': '2019-07-22 16:05:53',
+                            'name': 'Some, but not ALL of the knowledge',
+                            'notes': null,
+                            'paid': false,
+                            'times_clicked': 3,
+                            'upvotes': 1,
+                            'url': 'http://www.test.com'
+                        },
+                        'status': 'ok',
+                        'status_code': 200
+                      }
+        401:
+          $ref: '#/components/responses/Unauthorized'
+        422:
+          $ref: '#/components/responses/Invalid'
+      security:
+      - ApiKeyAuth: []
+  
 components:
   schemas:
+
     Error:
-      required:
-      - code
-      - message
       type: object
       properties:
-        code:
+        apiVersion: 
           type: string
-        message:
+          description: API version
+        data: 
+          type: object
+          description: A null data object
```

Comment:
I think that we probably don't want to specify in the docs that it's always null. At this point it certainly is, but perhaps someday we'll also include some data with some error response. I'm not sure why we would, but it's possible. Let's just say `A data object`